Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonCustomPaletteBase.cs

```diff
@@ -10,6 +10,12 @@
  */
 #endregion
 
+using System.Xml.Xsl;
+
+using Krypton.Toolkit.Properties;
+
+using static Krypton.Toolkit.PI;
+
```

Comment:
`using` statements should be moved to 'Global\GlobalDeclarations' (apart from 'static' ones) :).

Reply:
These are "Only" used here, So will not put into global usings